### PR TITLE
Allow resource string to be in english

### LINKED ISSUE #940: Add more test condition in CameraActivity.
**Actual Behaviour**

Currently, the camera activity tests only exposure and to take a photo.  

**Expected Behaviour**

Add more testing condition in the camera activity. 

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by Subhankar29**
Fix #940 

Changes: Testing more camera features:

- [x] Toggle button

- [x] Camera switch

Added resource file only to be in English, this would save space as automatically eliminates all resource files which in any other language than English

### CODE REVIEW

**Comment by anantprsd5** [comment]
@Subhankar29 Please resolve conflicts.

**Comment by Subhankar29** [comment]
@anantprsd5 Done.

File: app/src/androidTest/java/org/fossasia/phimpme/leafpic/activities/CameraActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        onView(
+                allOf(withId(R.id.popup), withContentDescription("Popup settings"), isDisplayed()));
+
+        onView(
+                allOf(withId(R.id.exposure), withContentDescription("Exposure"), isDisplayed()));
+
+        for (int i= 1; i<=5; i++) {
+            ViewInteraction appCompatImageButton3 = onView(
+                    allOf(withId(R.id.toggle_button), isDisplayed()));
+            appCompatImageButton3.perform(click());
```
Comment (pa1pal) [inline app/src/androidTest/java/org/fossasia/phimpme/leafpic/activities/CameraActivityTest.java]:
```
android.support.test.espresso.PerformException: Error performing 'single click - At Coordinates: 59, 176 and precision: 16, 16' on view 'Animations or transitions are enabled on the target device.
For more info check: http://goo.gl/qVu1yV

(with id: org.fossasia.phimpme:id/toggle_button and is displayed on the screen to the user)'.
at android.support.test.espresso.PerformException$Builder.build(PerformException.java:83)
at android.support.test.espresso.base.DefaultFailureHandler.getUserFriendlyError(DefaultFailureHandler.java:80)
at android.support.test.espresso.base.DefaultFailureHandler.handle(DefaultFailureHandler.java:56)
at android.support.test.espresso.ViewInteraction.runSynchronouslyOnUiThread(ViewInteraction.java:184)
at android.support.test.espresso.ViewInteraction.doPerform(ViewInteraction.java:115)
at android.support.test.espresso.ViewInteraction.perform(ViewInteraction.java:87)
at org.fossasia.phimpme.leafpic.activities.CameraActivityTest.cameraActivityTest(CameraActivityTest.java:72)
at org.fossasia.phimpme.leafpic.activities.CameraActivityTest.waitFor5Seconds(CameraActivityTest.java:44)
at java.lang.reflect.Method.invoke(Native Method)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
at android.support.test.internal.statement.UiThreadStatement.evaluate(UiThreadStatement.java:55)
at android.support.test.rule.ActivityTestRule$ActivityStatement.evaluate(ActivityTestRule.java:270)
at org.junit.rules.RunRules.evaluate(RunRules.java:20)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.junit.runners.Suite.runChild(Suite.java:128)
at org.junit.runners.Suite.runChild(Suite.java:27)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
```

Comment (pa1pal) [inline app/src/androidTest/java/org/fossasia/phimpme/leafpic/activities/CameraActivityTest.java]: @Subhankar29 Why there is loop?
no use. I think, its checking the view interactions only.

**Comment by anantprsd5** [comment]
@Subhankar29 There is some problem while opening the camera in the Travis emulator. Please add the "@ignore" annotation to stop the Travis from running the camera activity test or else the tests will always fail.

**Pull request merged**